web-ui/src/main/resources/catalog/views/default/directives/partials/mdactionmenu.html

```diff
@@ -54,9 +54,7 @@
       <!-- Retired metadata can't be published -->
       <li
         role="menuitem"
-        data-ng-if="md.canReview
-                      && md.draft != 'y'
-                      && md.mdStatus != 3"
+        data-ng-if="displayPublicationOption(md, user)"
```

Comment:
We also need to disable checkboxes in privileges panel

Reply:
ok, the backend throws an exception, but preventing to select the values in the UI looks better. I'll check that.

Reply:
@fxprunayre as it's possible to configure different user profile to publish and un-publish, not sure if the UI will be very clear to `Reviewer` users in such cases. For example, if reviewers can publish, but not un-publish, in the following case:

![privileges-dialog](https://user-images.githubusercontent.com/1695003/228838095-a93ea138-bb88-4005-bfeb-62ae022d3925.png)

For the special groups, the dialog should display the checked boxes as disabled and the unchecked boxes as enabled. Not sure if that can be more confusing for users?

Reply:
:) indeed. Can be confusing if you change those settings over time. Not sure what is best. It can be also quite complicated when updating privileges for a selection.

Maybe we should add labels for reviewers depending on the config  ? "Only administrator can publish/unpublish" and disable only if publish and unpublish is only allowed to admin ?

Reply:
If a reviewer and publication is allowed and unpublish is allowed (as currently)
* record not published = checkbox is enabled 
* record published = checkbox is enabled 
If a reviewer and publication is allowed and unpublish is not allowed 
* record not published = checkbox is enabled 
* record published = checkbox is disabled with tooltip 
If a reviewer and publication not allowed and unpublish is not allowed 
* record not published = checkbox is disabled with tooltip 
* record published = checkbox is disabled with tooltip  

What do you think ?

Reply:
Let me check about these changes.

Reply:
![image](https://user-images.githubusercontent.com/1701393/230336108-a25998d9-5ef9-49c0-8f57-061714296e68.png)